src/main/java/org/terracotta/statistics/StatisticsManager.java

```diff
@@ -50,7 +57,7 @@
   private static <T extends Enum<T>> OperationStatistic<T> createOperationStatistic(String name, Set<String> tags, Map<String, ? extends Object> properties, Class<T> resultType) {
     return new GeneralOperationStatistic<>(name, tags, properties, resultType);
   }
-  
+
```

Comment:
How come we have whitespace changes in other commits besides the whitespace changing one?